src/main/java/com/hp/gaia/mgs/amqp/AmqpManager.java

```diff
@@ -1,63 +0,0 @@
-package com.hp.gaia.mgs.amqp;
-
-import com.hp.gaia.mgs.services.PropertiesKeeperService;
-import com.rabbitmq.client.Channel;
-import com.rabbitmq.client.Connection;
-import com.rabbitmq.client.ConnectionFactory;
-
-import java.io.IOException;
-import java.util.concurrent.TimeoutException;
-
-/**
- * Created by belozovs on 6/30/2015.
- * Manage RabbitMQ connection for publish metrics
- */
-
-public class AmqpManager {
-
-    private static String MQ_SERVER_NAME, MQ_SERVER_USERNAME, MQ_SERVER_PASSWORD, QUEUE_NAME;
-    private static Integer MQ_SERVER_PORT;
-
-    private Channel channel = null;
```

Comment:
JAX-RS resources can be executed in a multithreaded env - hance you need to make sure your code aligned.
Channels for instance are not thread safe (https://www.rabbitmq.com/api-guide.html#channel-threads).